xml/System.Threading.Channels/ChannelReader`1.xml

```diff
@@ -85,7 +85,7 @@
         <ReturnType>System.Boolean</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Gets whether <see cref="M:System.Threading.Channels.ChannelReader`1.TryPeek(`0@)" /> is available for use on this <see cref="T:System.Threading.Channels.ChannelReader`1" /> instance.</summary>
         <value>To be added.</value>
```

Comment:
```suggestion
        <value><see langword="true" /> if peeking is supported by this channel instance; <see langword="false" /> otherwise.</value>
```